docs/source/markdown/podman-run.1.md

```diff
@@ -754,7 +754,7 @@ Security Options
 - **label=disable**: Turn off label separation for the container
 - **no-new-privileges**: Disable container processes from gaining additional privileges
 - **seccomp=unconfined**: Turn off seccomp confinement for the container
-- **seccomp**=_profile.json_:  Whitelisted syscalls seccomp JSON file to be used as a seccomp filter
+- **seccomp**=_profile.json_:  ALlowlisted syscalls seccomp JSON file to be used as a seccomp filter
```

Comment:
I think this would work better as `Allowed syscall list seccomp JSON file to be used as a seccomp filter`